week_1/css_fundamentals/Question4/style.css

```diff
@@ -1,57 +1,41 @@
-body{
+body {
     background-color: aliceblue;
 }
-.container{
-    background-color: white;
-    width:500px;
-    height:200px;
-    
 
+.container {
+    background-color: white;
+    width: 500px;
+    text-align: center;
+    margin: auto;
+    padding: 20px;
 }
-.nom{
+
+.nom {
     color: black;
-    text-align: center;
     font-family:'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
     font-weight: 100;
     font-size: 60px;
-    width: 600px;
-    height: 100px;
-    position: relative;
-    left: -150px;
-
 }
-.cool{
+
+.cool {
     background-color:yellow;
-    width: 100px;
-    height: 50px;
+    margin: auto;
+    width: fit-content;
+    padding: 10px 20px;
     font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
     font-weight:bolder;
     letter-spacing: 1px;
-    text-align: center; 
-    line-height: 50px;
-   border-radius: 12px;
-   position: relative;
-   top:-60px;
-   left:210px;
-  transform: rotate(-10deg)
-    
-
+    border-radius: 12px;
+    transform: rotate(-10deg) translate(100%, -10%);
 }
-p{
-    top:10px;   
-    text-align: center;
-     font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
-     font-weight: bold;
 
+p {
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    font-weight: bold;
 }
+
 .mp {
-    position: relative;
-    top: -38px; 
-    left: -100px; 
-  }
-  .pp{
-    position: relative;
-    top: -58px; 
-    left: -100px; 

```

Comment:
You shouldn't be centering elements like this.
Use text-align center, or margin-auto or flex